src/Form/Form.js

```diff
@@ -375,14 +375,29 @@ class Form {
         const mainInputType = getInputMainType(input)
         this.inputs[mainInputType].add(input)
 
+        // TODO: temporary pixel
+        const subtype = getInputSubtype(input)
+        if (subtype === 'emailAddress') {
+            this.addListener(input, 'pointerdown', () => {
+                this.device.firePixel({pixelName: 'incontext_eligible'})
+            }, {once: true})
+        }
+
         this.decorateInput(input)
 
         return this
     }
 
-    addListener (el, type, fn) {
-        el.addEventListener(type, fn)
-        this.listeners.add({el, type, fn})
+    /**
+     * Adds event listeners and keeps track of them for subsequent removal
+     * @param {HTMLElement} el
+     * @param {Event['type']} type
+     * @param {() => void} fn
+     * @param {AddEventListenerOptions} [opts]
```

Comment:
Just added the `opts` object so we can use this more flexibly as per the standard `addEventListener` specs.